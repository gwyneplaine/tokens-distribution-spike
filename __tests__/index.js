/// SNAPSHOT TESTS
const fs = require('fs');
const globby = require('globby');
const { execSync } = require('child_process');
const cases = require('jest-in-case');

const stripComments = (file) => {
  return file.replace(/\/\*([^*]|[\r\n]|(\*+([^*\/]|[\r\n])))*\*\/+/gi, '')
}

describe('on yarn build', () => {
  beforeAll(() => {
    execSync('yarn build');
  });
  afterAll(() => {
    rimraf.sync('packages/*/!(package.json)*/!(package.json)*');
  });

  cases('output files should match snapshots', opts => {
    const filePaths = globby.sync(opts.pattern, { ignore: ['CHANGELOG.md', 'package.json']});
    filePaths.sort().forEach(filePath => {
      const file = fs.readFileSync(filePath, 'utf-8').toString();
      expect(stripComments(file)).toMatchSnapshot();
    });
  }, [
    { pattern: './packages/design-tokens-css/*.css', name: 'css' },
    { pattern: './packages/design-tokens-scss/*.scss', name: 'scss' },
    { pattern: './packages/design-tokens-less/*.less', name: 'less' },
    { pattern: './packages/design-tokens-js/src/*.js', name: 'js' },
    { pattern: './packages/design-tokens-json/*/*', name: 'json' },
    { pattern: './packages/design-tokens-ts/src/*.ts', name: 'ts' },
  ]);
})
