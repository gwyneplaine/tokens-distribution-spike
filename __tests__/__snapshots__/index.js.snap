// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`on yarn build output files should match snapshots css 1`] = `
"

:root {
 --r_50: #FFEBE6;
 --r_75: #FFBDAD;
 --r_100: #FF8F73;
 --r_200: #FF7452;
 --r_300: #FF5630;
 --r_400: #DE350B;
 --r_500: #BF2600;
 --y_50: #FFFAE6;
 --y_75: #FFF0B3;
 --y_100: #FFE380;
 --y_200: #FFC400;
 --y_300: #FFAB00;
 --y_400: #FF991F;
 --y_500: #FF8B00;
 --g_50: #E3FCEF;
 --g_75: #ABF5D1;
 --g_100: #79F2C0;
 --g_200: #57D9A3;
 --g_300: #36B37E;
 --g_400: #00875A;
 --g_500: #006644;
 --b_50: #DEEBFF;
 --b_75: #B3D4FF;
 --b_100: #4C9AFF;
 --b_200: #2684FF;
 --b_300: #0065FF;
 --b_400: #0052CC;
 --b_500: #0747A6;
 --p_50: #EAE6FF;
 --p_75: #C0B6F2;
 --p_100: #998DD9;
 --p_200: #8777D9;
 --p_300: #6554C0;
 --p_400: #5243AA;
 --p_500: #403294;
 --t_50: #E6FCFF;
 --t_75: #B3F5FF;
 --t_100: #79E2F2;
 --t_200: #00C7E6;
 --t_300: #00B8D9;
 --t_400: #00A3BF;
 --t_500: #008DA6;
 --n_0: #FFFFFF;
 --n_10: #FAFBFC;
 --n_20: #F4F5F7;
 --n_30: #EBECF0;
 --n_40: #DFE1E6;
 --n_50: #C1C7D0;
 --n_60: #B3BAC5;
 --n_70: #A5ADBA;
 --n_80: #97A0AF;
 --n_90: #8993A4;
 --n_100: #7A869A;
 --n_200: #6B778C;
 --n_300: #5E6C84;
 --n_400: #505F79;
 --n_500: #42526E;
 --n_600: #344563;
 --n_700: #253858;
 --n_800: #172B4D;
 --n_900: #091E42;
}
"
`;

exports[`on yarn build output files should match snapshots css 2`] = `
"

:root {
 --r_50: #FFEBE6;
 --r_75: #FFBDAD;
 --r_100: #FF8F73;
 --r_200: #FF7452;
 --r_300: #FF5630;
 --r_400: #DE350B;
 --r_500: #BF2600;
 --y_50: #FFFAE6;
 --y_75: #FFF0B3;
 --y_100: #FFE380;
 --y_200: #FFC400;
 --y_300: #FFAB00;
 --y_400: #FF991F;
 --y_500: #FF8B00;
 --g_50: #E3FCEF;
 --g_75: #ABF5D1;
 --g_100: #79F2C0;
 --g_200: #57D9A3;
 --g_300: #36B37E;
 --g_400: #00875A;
 --g_500: #006644;
 --b_50: #DEEBFF;
 --b_75: #B3D4FF;
 --b_100: #4C9AFF;
 --b_200: #2684FF;
 --b_300: #0065FF;
 --b_400: #0052CC;
 --b_500: #0747A6;
 --p_50: #EAE6FF;
 --p_75: #C0B6F2;
 --p_100: #998DD9;
 --p_200: #8777D9;
 --p_300: #6554C0;
 --p_400: #5243AA;
 --p_500: #403294;
 --t_50: #E6FCFF;
 --t_75: #B3F5FF;
 --t_100: #79E2F2;
 --t_200: #00C7E6;
 --t_300: #00B8D9;
 --t_400: #00A3BF;
 --t_500: #008DA6;
 --n_0: #FFFFFF;
 --n_10: #FAFBFC;
 --n_20: #F4F5F7;
 --n_30: #EBECF0;
 --n_40: #DFE1E6;
 --n_50: #C1C7D0;
 --n_60: #B3BAC5;
 --n_70: #A5ADBA;
 --n_80: #97A0AF;
 --n_90: #8993A4;
 --n_100: #7A869A;
 --n_200: #6B778C;
 --n_300: #5E6C84;
 --n_400: #505F79;
 --n_500: #42526E;
 --n_600: #344563;
 --n_700: #253858;
 --n_800: #172B4D;
 --n_900: #091E42;
 --layer_100: 100;
 --layer_300: 300;
 --layer_200: 200;
 --layer_400: 400;
 --layer_500: 500;
 --layer_510: 510;
 --layer_600: 600;
 --layer_700: 700;
 --layer_800: 800;
 --grid_0_25_x: 2;
 --grid_0_5_x: 4;
 --grid_1_x: 8;
}
"
`;

exports[`on yarn build output files should match snapshots css 3`] = `
"

:root {
 --layer_100: 100;
 --layer_300: 300;
 --layer_200: 200;
 --layer_400: 400;
 --layer_500: 500;
 --layer_510: 510;
 --layer_600: 600;
 --layer_700: 700;
 --layer_800: 800;
}
"
`;

exports[`on yarn build output files should match snapshots css 4`] = `
"

:root {
 --grid_0_25_x: 2;
 --grid_0_5_x: 4;
 --grid_1_x: 8;
}
"
`;

exports[`on yarn build output files should match snapshots js 1`] = `
"

export const  R50 = \\"#FFEBE6\\";
export const  R75 = \\"#FFBDAD\\";
export const  R100 = \\"#FF8F73\\";
export const  R200 = \\"#FF7452\\";
export const  R300 = \\"#FF5630\\";
export const  R400 = \\"#DE350B\\";
export const  R500 = \\"#BF2600\\";
export const  Y50 = \\"#FFFAE6\\";
export const  Y75 = \\"#FFF0B3\\";
export const  Y100 = \\"#FFE380\\";
export const  Y200 = \\"#FFC400\\";
export const  Y300 = \\"#FFAB00\\";
export const  Y400 = \\"#FF991F\\";
export const  Y500 = \\"#FF8B00\\";
export const  G50 = \\"#E3FCEF\\";
export const  G75 = \\"#ABF5D1\\";
export const  G100 = \\"#79F2C0\\";
export const  G200 = \\"#57D9A3\\";
export const  G300 = \\"#36B37E\\";
export const  G400 = \\"#00875A\\";
export const  G500 = \\"#006644\\";
export const  B50 = \\"#DEEBFF\\";
export const  B75 = \\"#B3D4FF\\";
export const  B100 = \\"#4C9AFF\\";
export const  B200 = \\"#2684FF\\";
export const  B300 = \\"#0065FF\\";
export const  B400 = \\"#0052CC\\";
export const  B500 = \\"#0747A6\\";
export const  P50 = \\"#EAE6FF\\";
export const  P75 = \\"#C0B6F2\\";
export const  P100 = \\"#998DD9\\";
export const  P200 = \\"#8777D9\\";
export const  P300 = \\"#6554C0\\";
export const  P400 = \\"#5243AA\\";
export const  P500 = \\"#403294\\";
export const  T50 = \\"#E6FCFF\\";
export const  T75 = \\"#B3F5FF\\";
export const  T100 = \\"#79E2F2\\";
export const  T200 = \\"#00C7E6\\";
export const  T300 = \\"#00B8D9\\";
export const  T400 = \\"#00A3BF\\";
export const  T500 = \\"#008DA6\\";
export const  N0 = \\"#FFFFFF\\";
export const  N10 = \\"#FAFBFC\\";
export const  N20 = \\"#F4F5F7\\";
export const  N30 = \\"#EBECF0\\";
export const  N40 = \\"#DFE1E6\\";
export const  N50 = \\"#C1C7D0\\";
export const  N60 = \\"#B3BAC5\\";
export const  N70 = \\"#A5ADBA\\";
export const  N80 = \\"#97A0AF\\";
export const  N90 = \\"#8993A4\\";
export const  N100 = \\"#7A869A\\";
export const  N200 = \\"#6B778C\\";
export const  N300 = \\"#5E6C84\\";
export const  N400 = \\"#505F79\\";
export const  N500 = \\"#42526E\\";
export const  N600 = \\"#344563\\";
export const  N700 = \\"#253858\\";
export const  N800 = \\"#172B4D\\";
export const  N900 = \\"#091E42\\";"
`;

exports[`on yarn build output files should match snapshots js 2`] = `
"export * from './color.js';
export * from './layers.js';
export * from './spacing.js';"
`;

exports[`on yarn build output files should match snapshots js 3`] = `
"

export const layer100 = 100;
export const layer300 = 300;
export const layer200 = 200;
export const layer400 = 400;
export const layer500 = 500;
export const layer510 = 510;
export const layer600 = 600;
export const layer700 = 700;
export const layer800 = 800;"
`;

exports[`on yarn build output files should match snapshots js 4`] = `
"

export const grid025X = 2;
export const grid05X = 4;
export const grid1X = 8;"
`;

exports[`on yarn build output files should match snapshots json 1`] = `
"{
  \\"r_50\\": \\"#FFEBE6\\",
  \\"r_75\\": \\"#FFBDAD\\",
  \\"r_100\\": \\"#FF8F73\\",
  \\"r_200\\": \\"#FF7452\\",
  \\"r_300\\": \\"#FF5630\\",
  \\"r_400\\": \\"#DE350B\\",
  \\"r_500\\": \\"#BF2600\\",
  \\"y_50\\": \\"#FFFAE6\\",
  \\"y_75\\": \\"#FFF0B3\\",
  \\"y_100\\": \\"#FFE380\\",
  \\"y_200\\": \\"#FFC400\\",
  \\"y_300\\": \\"#FFAB00\\",
  \\"y_400\\": \\"#FF991F\\",
  \\"y_500\\": \\"#FF8B00\\",
  \\"g_50\\": \\"#E3FCEF\\",
  \\"g_75\\": \\"#ABF5D1\\",
  \\"g_100\\": \\"#79F2C0\\",
  \\"g_200\\": \\"#57D9A3\\",
  \\"g_300\\": \\"#36B37E\\",
  \\"g_400\\": \\"#00875A\\",
  \\"g_500\\": \\"#006644\\",
  \\"b_50\\": \\"#DEEBFF\\",
  \\"b_75\\": \\"#B3D4FF\\",
  \\"b_100\\": \\"#4C9AFF\\",
  \\"b_200\\": \\"#2684FF\\",
  \\"b_300\\": \\"#0065FF\\",
  \\"b_400\\": \\"#0052CC\\",
  \\"b_500\\": \\"#0747A6\\",
  \\"p_50\\": \\"#EAE6FF\\",
  \\"p_75\\": \\"#C0B6F2\\",
  \\"p_100\\": \\"#998DD9\\",
  \\"p_200\\": \\"#8777D9\\",
  \\"p_300\\": \\"#6554C0\\",
  \\"p_400\\": \\"#5243AA\\",
  \\"p_500\\": \\"#403294\\",
  \\"t_50\\": \\"#E6FCFF\\",
  \\"t_75\\": \\"#B3F5FF\\",
  \\"t_100\\": \\"#79E2F2\\",
  \\"t_200\\": \\"#00C7E6\\",
  \\"t_300\\": \\"#00B8D9\\",
  \\"t_400\\": \\"#00A3BF\\",
  \\"t_500\\": \\"#008DA6\\",
  \\"n_0\\": \\"#FFFFFF\\",
  \\"n_10\\": \\"#FAFBFC\\",
  \\"n_20\\": \\"#F4F5F7\\",
  \\"n_30\\": \\"#EBECF0\\",
  \\"n_40\\": \\"#DFE1E6\\",
  \\"n_50\\": \\"#C1C7D0\\",
  \\"n_60\\": \\"#B3BAC5\\",
  \\"n_70\\": \\"#A5ADBA\\",
  \\"n_80\\": \\"#97A0AF\\",
  \\"n_90\\": \\"#8993A4\\",
  \\"n_100\\": \\"#7A869A\\",
  \\"n_200\\": \\"#6B778C\\",
  \\"n_300\\": \\"#5E6C84\\",
  \\"n_400\\": \\"#505F79\\",
  \\"n_500\\": \\"#42526E\\",
  \\"n_600\\": \\"#344563\\",
  \\"n_700\\": \\"#253858\\",
  \\"n_800\\": \\"#172B4D\\",
  \\"n_900\\": \\"#091E42\\"
}"
`;

exports[`on yarn build output files should match snapshots json 2`] = `
"{
  \\"r_50\\": \\"#FFEBE6\\",
  \\"r_75\\": \\"#FFBDAD\\",
  \\"r_100\\": \\"#FF8F73\\",
  \\"r_200\\": \\"#FF7452\\",
  \\"r_300\\": \\"#FF5630\\",
  \\"r_400\\": \\"#DE350B\\",
  \\"r_500\\": \\"#BF2600\\",
  \\"y_50\\": \\"#FFFAE6\\",
  \\"y_75\\": \\"#FFF0B3\\",
  \\"y_100\\": \\"#FFE380\\",
  \\"y_200\\": \\"#FFC400\\",
  \\"y_300\\": \\"#FFAB00\\",
  \\"y_400\\": \\"#FF991F\\",
  \\"y_500\\": \\"#FF8B00\\",
  \\"g_50\\": \\"#E3FCEF\\",
  \\"g_75\\": \\"#ABF5D1\\",
  \\"g_100\\": \\"#79F2C0\\",
  \\"g_200\\": \\"#57D9A3\\",
  \\"g_300\\": \\"#36B37E\\",
  \\"g_400\\": \\"#00875A\\",
  \\"g_500\\": \\"#006644\\",
  \\"b_50\\": \\"#DEEBFF\\",
  \\"b_75\\": \\"#B3D4FF\\",
  \\"b_100\\": \\"#4C9AFF\\",
  \\"b_200\\": \\"#2684FF\\",
  \\"b_300\\": \\"#0065FF\\",
  \\"b_400\\": \\"#0052CC\\",
  \\"b_500\\": \\"#0747A6\\",
  \\"p_50\\": \\"#EAE6FF\\",
  \\"p_75\\": \\"#C0B6F2\\",
  \\"p_100\\": \\"#998DD9\\",
  \\"p_200\\": \\"#8777D9\\",
  \\"p_300\\": \\"#6554C0\\",
  \\"p_400\\": \\"#5243AA\\",
  \\"p_500\\": \\"#403294\\",
  \\"t_50\\": \\"#E6FCFF\\",
  \\"t_75\\": \\"#B3F5FF\\",
  \\"t_100\\": \\"#79E2F2\\",
  \\"t_200\\": \\"#00C7E6\\",
  \\"t_300\\": \\"#00B8D9\\",
  \\"t_400\\": \\"#00A3BF\\",
  \\"t_500\\": \\"#008DA6\\",
  \\"n_0\\": \\"#FFFFFF\\",
  \\"n_10\\": \\"#FAFBFC\\",
  \\"n_20\\": \\"#F4F5F7\\",
  \\"n_30\\": \\"#EBECF0\\",
  \\"n_40\\": \\"#DFE1E6\\",
  \\"n_50\\": \\"#C1C7D0\\",
  \\"n_60\\": \\"#B3BAC5\\",
  \\"n_70\\": \\"#A5ADBA\\",
  \\"n_80\\": \\"#97A0AF\\",
  \\"n_90\\": \\"#8993A4\\",
  \\"n_100\\": \\"#7A869A\\",
  \\"n_200\\": \\"#6B778C\\",
  \\"n_300\\": \\"#5E6C84\\",
  \\"n_400\\": \\"#505F79\\",
  \\"n_500\\": \\"#42526E\\",
  \\"n_600\\": \\"#344563\\",
  \\"n_700\\": \\"#253858\\",
  \\"n_800\\": \\"#172B4D\\",
  \\"n_900\\": \\"#091E42\\",
  \\"layer_100\\": 100,
  \\"layer_300\\": 300,
  \\"layer_200\\": 200,
  \\"layer_400\\": 400,
  \\"layer_500\\": 500,
  \\"layer_510\\": 510,
  \\"layer_600\\": 600,
  \\"layer_700\\": 700,
  \\"layer_800\\": 800,
  \\"grid_0_25_x\\": 2,
  \\"grid_0_5_x\\": 4,
  \\"grid_1_x\\": 8
}"
`;

exports[`on yarn build output files should match snapshots json 3`] = `
"{
  \\"layer_100\\": 100,
  \\"layer_300\\": 300,
  \\"layer_200\\": 200,
  \\"layer_400\\": 400,
  \\"layer_500\\": 500,
  \\"layer_510\\": 510,
  \\"layer_600\\": 600,
  \\"layer_700\\": 700,
  \\"layer_800\\": 800
}"
`;

exports[`on yarn build output files should match snapshots json 4`] = `
"{
  \\"grid_0_25_x\\": 2,
  \\"grid_0_5_x\\": 4,
  \\"grid_1_x\\": 8
}"
`;

exports[`on yarn build output files should match snapshots less 1`] = `
"

@r_50: #FFEBE6;
@r_75: #FFBDAD;
@r_100: #FF8F73;
@r_200: #FF7452;
@r_300: #FF5630;
@r_400: #DE350B;
@r_500: #BF2600;
@y_50: #FFFAE6;
@y_75: #FFF0B3;
@y_100: #FFE380;
@y_200: #FFC400;
@y_300: #FFAB00;
@y_400: #FF991F;
@y_500: #FF8B00;
@g_50: #E3FCEF;
@g_75: #ABF5D1;
@g_100: #79F2C0;
@g_200: #57D9A3;
@g_300: #36B37E;
@g_400: #00875A;
@g_500: #006644;
@b_50: #DEEBFF;
@b_75: #B3D4FF;
@b_100: #4C9AFF;
@b_200: #2684FF;
@b_300: #0065FF;
@b_400: #0052CC;
@b_500: #0747A6;
@p_50: #EAE6FF;
@p_75: #C0B6F2;
@p_100: #998DD9;
@p_200: #8777D9;
@p_300: #6554C0;
@p_400: #5243AA;
@p_500: #403294;
@t_50: #E6FCFF;
@t_75: #B3F5FF;
@t_100: #79E2F2;
@t_200: #00C7E6;
@t_300: #00B8D9;
@t_400: #00A3BF;
@t_500: #008DA6;
@n_0: #FFFFFF;
@n_10: #FAFBFC;
@n_20: #F4F5F7;
@n_30: #EBECF0;
@n_40: #DFE1E6;
@n_50: #C1C7D0;
@n_60: #B3BAC5;
@n_70: #A5ADBA;
@n_80: #97A0AF;
@n_90: #8993A4;
@n_100: #7A869A;
@n_200: #6B778C;
@n_300: #5E6C84;
@n_400: #505F79;
@n_500: #42526E;
@n_600: #344563;
@n_700: #253858;
@n_800: #172B4D;
@n_900: #091E42;"
`;

exports[`on yarn build output files should match snapshots less 2`] = `
"@import './color.less';
@import './layers.less';
@import './spacing.less';"
`;

exports[`on yarn build output files should match snapshots less 3`] = `
"

@layer_100: 100;
@layer_300: 300;
@layer_200: 200;
@layer_400: 400;
@layer_500: 500;
@layer_510: 510;
@layer_600: 600;
@layer_700: 700;
@layer_800: 800;"
`;

exports[`on yarn build output files should match snapshots less 4`] = `
"

@grid_0_25_x: 2;
@grid_0_5_x: 4;
@grid_1_x: 8;"
`;

exports[`on yarn build output files should match snapshots scss 1`] = `
"

$r_50: #FFEBE6;
$r_75: #FFBDAD;
$r_100: #FF8F73;
$r_200: #FF7452;
$r_300: #FF5630;
$r_400: #DE350B;
$r_500: #BF2600;
$y_50: #FFFAE6;
$y_75: #FFF0B3;
$y_100: #FFE380;
$y_200: #FFC400;
$y_300: #FFAB00;
$y_400: #FF991F;
$y_500: #FF8B00;
$g_50: #E3FCEF;
$g_75: #ABF5D1;
$g_100: #79F2C0;
$g_200: #57D9A3;
$g_300: #36B37E;
$g_400: #00875A;
$g_500: #006644;
$b_50: #DEEBFF;
$b_75: #B3D4FF;
$b_100: #4C9AFF;
$b_200: #2684FF;
$b_300: #0065FF;
$b_400: #0052CC;
$b_500: #0747A6;
$p_50: #EAE6FF;
$p_75: #C0B6F2;
$p_100: #998DD9;
$p_200: #8777D9;
$p_300: #6554C0;
$p_400: #5243AA;
$p_500: #403294;
$t_50: #E6FCFF;
$t_75: #B3F5FF;
$t_100: #79E2F2;
$t_200: #00C7E6;
$t_300: #00B8D9;
$t_400: #00A3BF;
$t_500: #008DA6;
$n_0: #FFFFFF;
$n_10: #FAFBFC;
$n_20: #F4F5F7;
$n_30: #EBECF0;
$n_40: #DFE1E6;
$n_50: #C1C7D0;
$n_60: #B3BAC5;
$n_70: #A5ADBA;
$n_80: #97A0AF;
$n_90: #8993A4;
$n_100: #7A869A;
$n_200: #6B778C;
$n_300: #5E6C84;
$n_400: #505F79;
$n_500: #42526E;
$n_600: #344563;
$n_700: #253858;
$n_800: #172B4D;
$n_900: #091E42;"
`;

exports[`on yarn build output files should match snapshots scss 2`] = `
"@import './color.scss';
@import './layers.scss';
@import './spacing.scss';"
`;

exports[`on yarn build output files should match snapshots scss 3`] = `
"

$layer_100: 100;
$layer_300: 300;
$layer_200: 200;
$layer_400: 400;
$layer_500: 500;
$layer_510: 510;
$layer_600: 600;
$layer_700: 700;
$layer_800: 800;"
`;

exports[`on yarn build output files should match snapshots scss 4`] = `
"

$grid_0_25_x: 2;
$grid_0_5_x: 4;
$grid_1_x: 8;"
`;

exports[`on yarn build output files should match snapshots ts 1`] = `
"

export const  R50 = \\"#FFEBE6\\";
export const  R75 = \\"#FFBDAD\\";
export const  R100 = \\"#FF8F73\\";
export const  R200 = \\"#FF7452\\";
export const  R300 = \\"#FF5630\\";
export const  R400 = \\"#DE350B\\";
export const  R500 = \\"#BF2600\\";
export const  Y50 = \\"#FFFAE6\\";
export const  Y75 = \\"#FFF0B3\\";
export const  Y100 = \\"#FFE380\\";
export const  Y200 = \\"#FFC400\\";
export const  Y300 = \\"#FFAB00\\";
export const  Y400 = \\"#FF991F\\";
export const  Y500 = \\"#FF8B00\\";
export const  G50 = \\"#E3FCEF\\";
export const  G75 = \\"#ABF5D1\\";
export const  G100 = \\"#79F2C0\\";
export const  G200 = \\"#57D9A3\\";
export const  G300 = \\"#36B37E\\";
export const  G400 = \\"#00875A\\";
export const  G500 = \\"#006644\\";
export const  B50 = \\"#DEEBFF\\";
export const  B75 = \\"#B3D4FF\\";
export const  B100 = \\"#4C9AFF\\";
export const  B200 = \\"#2684FF\\";
export const  B300 = \\"#0065FF\\";
export const  B400 = \\"#0052CC\\";
export const  B500 = \\"#0747A6\\";
export const  P50 = \\"#EAE6FF\\";
export const  P75 = \\"#C0B6F2\\";
export const  P100 = \\"#998DD9\\";
export const  P200 = \\"#8777D9\\";
export const  P300 = \\"#6554C0\\";
export const  P400 = \\"#5243AA\\";
export const  P500 = \\"#403294\\";
export const  T50 = \\"#E6FCFF\\";
export const  T75 = \\"#B3F5FF\\";
export const  T100 = \\"#79E2F2\\";
export const  T200 = \\"#00C7E6\\";
export const  T300 = \\"#00B8D9\\";
export const  T400 = \\"#00A3BF\\";
export const  T500 = \\"#008DA6\\";
export const  N0 = \\"#FFFFFF\\";
export const  N10 = \\"#FAFBFC\\";
export const  N20 = \\"#F4F5F7\\";
export const  N30 = \\"#EBECF0\\";
export const  N40 = \\"#DFE1E6\\";
export const  N50 = \\"#C1C7D0\\";
export const  N60 = \\"#B3BAC5\\";
export const  N70 = \\"#A5ADBA\\";
export const  N80 = \\"#97A0AF\\";
export const  N90 = \\"#8993A4\\";
export const  N100 = \\"#7A869A\\";
export const  N200 = \\"#6B778C\\";
export const  N300 = \\"#5E6C84\\";
export const  N400 = \\"#505F79\\";
export const  N500 = \\"#42526E\\";
export const  N600 = \\"#344563\\";
export const  N700 = \\"#253858\\";
export const  N800 = \\"#172B4D\\";
export const  N900 = \\"#091E42\\";"
`;

exports[`on yarn build output files should match snapshots ts 2`] = `
"

export const  R50 = \\"#FFEBE6\\";
export const  R75 = \\"#FFBDAD\\";
export const  R100 = \\"#FF8F73\\";
export const  R200 = \\"#FF7452\\";
export const  R300 = \\"#FF5630\\";
export const  R400 = \\"#DE350B\\";
export const  R500 = \\"#BF2600\\";
export const  Y50 = \\"#FFFAE6\\";
export const  Y75 = \\"#FFF0B3\\";
export const  Y100 = \\"#FFE380\\";
export const  Y200 = \\"#FFC400\\";
export const  Y300 = \\"#FFAB00\\";
export const  Y400 = \\"#FF991F\\";
export const  Y500 = \\"#FF8B00\\";
export const  G50 = \\"#E3FCEF\\";
export const  G75 = \\"#ABF5D1\\";
export const  G100 = \\"#79F2C0\\";
export const  G200 = \\"#57D9A3\\";
export const  G300 = \\"#36B37E\\";
export const  G400 = \\"#00875A\\";
export const  G500 = \\"#006644\\";
export const  B50 = \\"#DEEBFF\\";
export const  B75 = \\"#B3D4FF\\";
export const  B100 = \\"#4C9AFF\\";
export const  B200 = \\"#2684FF\\";
export const  B300 = \\"#0065FF\\";
export const  B400 = \\"#0052CC\\";
export const  B500 = \\"#0747A6\\";
export const  P50 = \\"#EAE6FF\\";
export const  P75 = \\"#C0B6F2\\";
export const  P100 = \\"#998DD9\\";
export const  P200 = \\"#8777D9\\";
export const  P300 = \\"#6554C0\\";
export const  P400 = \\"#5243AA\\";
export const  P500 = \\"#403294\\";
export const  T50 = \\"#E6FCFF\\";
export const  T75 = \\"#B3F5FF\\";
export const  T100 = \\"#79E2F2\\";
export const  T200 = \\"#00C7E6\\";
export const  T300 = \\"#00B8D9\\";
export const  T400 = \\"#00A3BF\\";
export const  T500 = \\"#008DA6\\";
export const  N0 = \\"#FFFFFF\\";
export const  N10 = \\"#FAFBFC\\";
export const  N20 = \\"#F4F5F7\\";
export const  N30 = \\"#EBECF0\\";
export const  N40 = \\"#DFE1E6\\";
export const  N50 = \\"#C1C7D0\\";
export const  N60 = \\"#B3BAC5\\";
export const  N70 = \\"#A5ADBA\\";
export const  N80 = \\"#97A0AF\\";
export const  N90 = \\"#8993A4\\";
export const  N100 = \\"#7A869A\\";
export const  N200 = \\"#6B778C\\";
export const  N300 = \\"#5E6C84\\";
export const  N400 = \\"#505F79\\";
export const  N500 = \\"#42526E\\";
export const  N600 = \\"#344563\\";
export const  N700 = \\"#253858\\";
export const  N800 = \\"#172B4D\\";
export const  N900 = \\"#091E42\\";
export const layer100 = 100;
export const layer300 = 300;
export const layer200 = 200;
export const layer400 = 400;
export const layer500 = 500;
export const layer510 = 510;
export const layer600 = 600;
export const layer700 = 700;
export const layer800 = 800;
export const grid025X = 2;
export const grid05X = 4;
export const grid1X = 8;"
`;

exports[`on yarn build output files should match snapshots ts 3`] = `
"

export const layer100 = 100;
export const layer300 = 300;
export const layer200 = 200;
export const layer400 = 400;
export const layer500 = 500;
export const layer510 = 510;
export const layer600 = 600;
export const layer700 = 700;
export const layer800 = 800;"
`;

exports[`on yarn build output files should match snapshots ts 4`] = `
"

export const grid025X = 2;
export const grid05X = 4;
export const grid1X = 8;"
`;
